crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -752,10 +752,33 @@ new_key_type! {
     pub struct SystemSetKey;
 }
 
+/// Systems and system sets in a [`ScheduleGraph`] which have not been
+/// initialized yet.
```

Comment:
Not a big deal, but this phrasing feels slightly off.  It ought to be singular, and system sets themselves aren't really initialized.  Maybe something like 

```suggestion
/// A node in a [`ScheduleGraph`] with a system or conditions that have not been
/// initialized yet.
```

?